src/screens/ProfileScreen/index.js

```diff
@@ -719,24 +717,24 @@ const ProfileScreen = ({route}) => {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: colors.white,
+    backgroundColor: COLORS.white,
     height: '100%'
   },
   content: {
     flexDirection: 'column',
     paddingHorizontal: 20,
-    backgroundColor: colors.white,
+    backgroundColor: COLORS.white,
     marginTop: 14
   },
   dummyItem: (heightItem) => ({
     height: heightItem,
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
   }),
-  postText: (isActive) => ({
+  postText: (isActive, isSigned) => ({
     fontFamily: isActive ? fonts.inter[600] : fonts.inter[400],
     fontSize: 14,
     lineHeight: 17,
-    color: isActive ? colors.bondi_blue : colors.blackgrey,
+    color: isActive ? (isSigned ? COLORS.blue : COLORS.holyTosca) : COLORS.blackgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in color constants from `colors` to `COLORS` are consistent with the import statement change at the top of the file. This ensures that the correct color values are used throughout the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->